### PR TITLE
Signals dash generator: store a longer time series by using covidcast/coverage endpoint

### PR DESCRIPTION
**Pull request by krivard**
**Prerequisites**:

- [x] Unless it is a documentation hotfix it should be merged against the `dev` branch
- [x] Branch is up-to-date with the branch to be merged with, i.e. `dev`
- [x] Build is successful
- [x] Code is cleaned up and formatted

### Summary

Most of this PR is for switching the signals dashboard data generator over to the `covidcast/coverage` endpoint instead of pulling the regional data and counting sizes in a massive groupby. Since `coverage` isn't available natively in the covidcast library (and won't be; that's all going into the new delphi_epidata library/ies) we have to do this by constructing the URL, calling `pandas.read_csv`, and parsing the `time_value` column.

With that change, it makes it less of a big deal to double the length of the coverage time series -- the true goal of this PR.

### CODE REVIEW

File: src/acquisition/covidcast/signal_dash_data_generator.py [hunk 1]
```diff
@@ -17,7 +18,9 @@
 from delphi.epidata.acquisition.covidcast.logger import get_structured_logger
 
 
-LOOKBACK_DAYS_FOR_COVERAGE = 28
+LOOKBACK_DAYS_FOR_COVERAGE = 52
```
Comment (melange396) [inline src/acquisition/covidcast/signal_dash_data_generator.py:21]: out of curiousity, why 52?

**Review by melange396 — APPROVED** [review]
LGTM

File: src/acquisition/covidcast/signal_dash_data_generator.py (hunk 1 above)
Comment (krivard) [reply src/acquisition/covidcast/signal_dash_data_generator.py:21]: uhhh because I can't multiple 28 by 2, apparently

Comment (krivard) [inline src/acquisition/covidcast/signal_dash_data_generator.py:21]:
```suggestion
LOOKBACK_DAYS_FOR_COVERAGE = 56
```

**Pull request merged**